xla/tools/hlo_extractor.cc

```diff
@@ -42,7 +42,6 @@ limitations under the License.
 #include "xla/service/hlo_verifier.h"
 #include "xla/shape.h"
 #include "xla/shape_util.h"
-#include "xla/tests/test_utils.h"
```

Comment:
Please add xla/literal_util.h include instead. Also the corresponding BUILD dependency.

Reply:
That is already included in line number 38 and the dependency also exists already.

Reply:
It is already added in line 39. And the dependency also exists.

Reply:
Ah, sorry, I should have expanded the view to see it.